### DISCUSSION TITLE
Does often reading of Data Objects affect Optiga?

### DISCUSSION THREAD

**Discussion by SoftAvocado** [Infineon/optiga-trust-m#112] [General]
Hello!

I'm using Optiga Trust M V3 in a device. While the device is on, every 2 seconds a Data Object (Security Event Counter - 0xE0C5) is being read. So basically Optiga Trust M is being triggered for reading every 2 seconds for an unlimited amount of time. Can you please tell me if such actions can affect Optiga in any way, e.g. reduce the chip's lifespan?

**Comment by ShaikKarishma0** [comment]
Hi @SoftAvocado,

Reading a data object does not affect the chip's lifespan. But, writing to the data object does. As mentioned in the [solution reference manual](https://github.com/Infineon/optiga-trust-m/blob/develop/documents/OPTIGA%E2%84%A2%20Trust%20M%20Solution%20Reference%20Manual.md), the maximum number of times the Security Event Counter can be updated is 1.6M times.
Please refer [this](https://community.infineon.com/t5/Blogs/How-does-OPTIGA-Trust-M-prevent-attacks/ba-p/363652) blog on security monitor for more details on how the Security Event Counter is updated in NVM.

**Reply by SoftAvocado** [reply]
Got it, thank you!
